jlm/rvsdg/node.cpp

```diff
@@ -337,4 +349,58 @@ normalize(jlm::rvsdg::node * node)
   return nf->normalize_node(node);
 }
 
+/**
+  \page def_use_inspection Inspecting the graph and dispatching of different operations
+
+  When inspecting the graph for analysis it is necessary to identify
+  different nodes/operations and structures. Depending on the direction,
+  the two fundamental questions of interest are:
+
+  - what is the origin of a value, what operation is computing it?
+  - what are the users of a particular value, what operations depend on it?
+
+  This requires resolving the type of operation a specific \ref rvsdg::input
+  or \ref rvsdg::output belong to. Every \p output is one of the following:
+
+  - the output of a node representing an operation
+  - the entry argument into a region
+
+  Likewise, every \p input is one of the following:
+
+  - the input of a node representing an operation
+  - the exit result of a region
+
+  Analysis code can determine which of the two is the case using
+  \ref rvsdg::output::GetOwner and \ref rvsdg::input::GetOwner, respectively,
+  and then branch deeper based on its results. For convenience, code
+  can more directly check immediately for the specific kinds of nodes
+  involved using
```

Comment:
`involved using ...` Something is missing here.

Reply:
fixed